testing/tutorials/readme.md

```diff
@@ -0,0 +1,5 @@
+[Tutorial 1 2D Segmentation UNet ![Open n Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ivadomed/ivadomed/blob/yd/incorporating_colab_tutorials/testing/tutorials/tutorial_1_2d_segmentation_unet.ipynb)
```

Comment:
Just a minor type here! (I think it doesn't matter as the badge shows it correctly)

Reply:
Sharp eyes! Thanks!